eng/scripts/Language-Settings.ps1

```diff
@@ -15,25 +15,16 @@ function Get-dotnet-PackageInfoFromRepo ($pkgPath, $serviceDirectory, $pkgName)
     return $null
   }
 
-  if ($pkgName)
+  $projectPaths = (Resolve-Path (Join-Path $projDirPath "*.csproj")).path
+  if ($projectPaths -is [array] -and $projectPaths.Count -gt 1)
   {
-    $projectPath = Join-Path $projDirPath "$pkgName.csproj"
+    LogWarning "There is more than on csproj file in the projectpath/src directory. First project picked."
+    $projectPath = $projectPaths[0]
   }
-  else
-  {
-    $projectPaths = (Resolve-Path (Join-Path $projDirPath "*.csproj")).path
-    if ($projectPaths.Count -gt 1)
-    {
-      LogWarning "There is more than on csproj file in the projectpath/src directory. First project picked."
-      $projectPath = $projectPaths[0]
-    }
-    else {
-      $projectPath = $projectPaths
-    }
+  else {
+    $projectPath = $projectPaths -Join ""
```

Comment:
```suggestion
    $projectPath = $projectPaths
```

Reply:
The issue here seems to be that `Set-StrictMode` does not allow the Count property on a String.
`$projectPaths = (Resolve-Path (Join-Path $projDirPath "*.csproj")).path` returns array or string depending on how many paths it resolves to. I added to join to make sure we always end up with a single string.